deployer/README.md

```diff
@@ -264,6 +268,12 @@ This sub-command can be used to generate the resource allocation choices for giv
 ##### `generate resource-allocation choices`
 This generates a custom number of resource allocation choices for a certain instance type, depending on a certain chosen strategy that can be used in the profile list of a hub.
 
+##### `generate resource-allocation daemonset-overhead`
+Updates `daemonset_overhead.yaml` with an individual cluster's DaemonSets with running pods combined requests of CPU and memory, excluding GPU related DaemonSets.
+
+##### `generate resource-allocation instance-capacity`
+Updates `instance_capacity.yaml` with instance types' total and allocatable capacity reported by `kubectl get node`.
```

Comment:
From what I understand this command requires a cluster, and it uses that cluster to get the running nodes and updates the file with info about that or those running nodes, right?

From this description I understand that it updates all instance types and I wouldn't expect the command to require a cluster.

So maybe rephrase to:
```suggestion
This script updates the information in the `instance_capacity.yaml` file about the instance types found running in a given cluster.

This file contains the total and allocatable capacity as reported by `kubectl get node` of the instance types'  used across our clusters.
```

Reply:
I'll add a commit to refine this part similar to your suggestion, both here and in the .py file where this text is also duplicated as a docstring!